src/main/java/seedu/financialplanner/list/Budget.java

```diff
@@ -31,6 +33,16 @@ public static double getCurrentBudget() {
         return currentBudget;
     }
 
+    public static String getCurrentBudgetString() {
+        DecimalFormat decimalFormat = new DecimalFormat("####0.00");
+        return decimalFormat.format(Cashflow.round(currentBudget, 2));
```

Comment:
good job on using rounding function to make the value more accurate when displayed in 2d.p.